src/rgw/rgw_rados.cc

```diff
@@ -110,6 +110,7 @@ static RGWObjCategory main_category = RGWObjCategory::Main;
 
 const std::string MP_META_SUFFIX = ".meta";
 
+constexpr int64_t RGWRados::Bucket::List::bucket_list_objects_absolute_max;
```

Comment:
nit: static constexpr variables don't require this separate definition

Reply:
That's what I thought, but then I got a linking error. After some fiddling ended up with this. But I just removed it and everything compiled and linked just fine. So it must have been a misattribution on my part.